tuxemon/states/combat/combat.py

```diff
@@ -176,13 +178,13 @@ def get(
 
     @staticmethod
     def load_technique_animation(
-        technique: Union[Technique, Condition], is_flipped: bool
+        technique: Union[Technique, Condition, Item], is_flipped: bool
     ) -> Optional[Sprite]:
         """
-        Return animated sprite from a technique.
+        Return animated sprite from a technique, condition or item.
```

Comment:
Interesting, so looks like all these methods are not supporting only technique anymore but also condition or item, I was confused about the name.

I think everything should be renamed (argument names, method names etc.). Yes I know, "big refactoring", but it would be really confusing to have a method named `load_technique_animation`.... That in fact doesn't load only technique animations...